src/services/players/player_service.hpp

```diff
@@ -6,8 +6,8 @@ namespace big
 	class player;
 
 	using player_ptr = std::shared_ptr<player>;
-	using player_entry = std::pair<std::string, player_ptr>;
-	using players = std::map<std::string, player_ptr>;
+	using player_entry = std::pair<std::uint8_t, player_ptr>;
+	using players = std::map<std::uint8_t, player_ptr>;
```

Comment:
This change will remove the alphabetical ordering on the player list, I looked around but did not see you do anything else to order it alphabetically.